src/biogeochem/AtmCarbonIsotopeStreamType.F90

```diff
@@ -0,0 +1,193 @@
+module AtmCarbonIsotopeStreamType
```

Comment:
This puts both C13 and C14 in the same file -- it could be separated into two files.